frontend/src/Editor/ManageAppUsers.jsx

```diff
@@ -246,7 +251,9 @@ class ManageAppUsersComponent extends React.Component {
                     <div className="input-group">
                       <Textarea
                         disabled
-                        className={`input-with-icon ${this.props.darkMode && 'text-light'}`}
+                        className={`input-with-icon ${this.props.darkMode && 'bg-dark'} ${
+                          this.props.darkMode && 'text-light'
```

Comment:
As you are repeating the destructuring `this.props.darkMode` more than two times. You should destructure the `darkMode` outside JSX and use it.

Reply:
Okay with do that

Reply:
> As you are repeating the destructuring `this.props.darkMode` more than two times. You should destructure the `darkMode` outside JSX and use it.

Hey! done that